test/middleware/express.test.ts

```diff
@@ -34,20 +34,20 @@ const schema: GraphQLSchema = buildSchema(`
                 interface Character {
                     id: ID!
                     name: String!
-                    friends: [Character]
+                    friends: [Character] @listCost(cost: 10)
```

Comment:
I think its worth using differing lists costs on line 37, 44, 50 to ensure the correct cost is used?

Reply:
Actually, it doesn't like these are used in these tests

Reply:
I'm not sure what you mean by this.